web_edit_user_filter/static/src/js/backend.js

```diff
@@ -174,7 +187,10 @@ odoo.define("web_edit_user_filter/static/src/js/backend.js", function (require)
                     } else {
                         self.model.dispatch("toggleFilter", facetIdEl);
                     }
-                } else if (type === "groupBy") {
+                } else if (type === "field") {
+                    self.model.dispatch("toggleFilter", facetIdEl);
+                }
+                else if (type === "groupBy") {
                     self.model.dispatch("toggleFilter", facetIdEl);
```

Comment:
**suggestion**: We could merge these checks into a single one

```suggestion
                } else if (type === "groupBy" || type === "field") {
                    self.model.dispatch("toggleFilter", facetIdEl);
```

Reply:
Done